### ISSUE TITLE
problem in panel/code/ 

### ISSUE THREAD

**Issue by Tonenz16** [ProTechEx/IPTV-Panel#2]
hello, everything works for me! I only get 2 errors
the first is that I can't add the style to the admin panel. 
the second is that in panel / code / all files give me the error: fatal error; call to undefined function db_query()
in practice, when a reseller creates a line, it does not remove any credit. how could I fix it? has anyone solved it?

**Comment by LuxNetMedia-com** [comment]
> hello, everything works for me! I only get 2 errors the first is that I can't add the style to the admin panel. the second is that in panel / code / all files give me the error: fatal error; call to undefined function db_query() in practice, when a reseller creates a line, it does not remove any credit. how could I fix it? has anyone solved it?

hi tonenz16, did you get it to work?

**Comment by Tonenz16** [comment]
Hi Lux, it's a very old project, I abandoned it years ago, about 3/4 years have passed, I remembered having fixed it yes, basically everything is based on the database query and the credits will work, check again you have to configure the database well, if I'm not mistaken this project is in php.

**Comment by LuxNetMedia-com** [comment]
thank u. i am fixing it now because it´s not working with php 8. or do you have another working panel?

________________________________
Von: Tonenz16 ***@***.***>
Gesendet: Montag, 8. September 2025 11:03
An: ProTechEx/IPTV-Panel ***@***.***>
Cc: LuxNetMedia-com ***@***.***>; Comment ***@***.***>
Betreff: Re: [ProTechEx/IPTV-Panel] problem in panel/code/ (#2)

[https://avatars.githubusercontent.com/u/81432984?s=20&v=4]Tonenz16 left a comment (ProTechEx/IPTV-Panel#2)<https://github.com/ProTechEx/IPTV-Panel/issues/2#issuecomment-3265310706>

Hi Lux, it's a very old project, I abandoned it years ago, about 3/4 years have passed, I remembered having fixed it yes, basically everything is based on the database query and the credits will work, check again you have to configure the database well, if I'm not mistaken this project is in php.

**Comment by Tonenz16** [comment]
No, I changed programming language, I switched to Python, if you want I can develop it custom, at the moment I don't have a CMS already created, sorry.
